reporting_endpoint.py

```diff
@@ -17,16 +17,16 @@
 reporting = Blueprint('reporting', __name__, template_folder='templates/reporting')
 
 ### FREQUENTLY USE DATES
-TODAY = lambda: date.today() # pylint: disable=unnecessary-lambda
-YESTERDAY = lambda: date.today() - timedelta(days=1)
-TOMORROW = lambda: date.today() + timedelta(days=1)
-WEEKDAY = lambda: date.today().weekday()
-MONDAY = lambda: date.today() - timedelta(days=date.today().weekday())
-NEXT_MONDAY = lambda: date.today() + timedelta(days=-date.today().weekday(), weeks=1)
-FIRST_DAY_CURRENT_MONTH = lambda: date.today().replace(day=1)
-FIRST_DAY_NEXT_MONTH = lambda: date.today().replace(day=1) + relativedelta(months=+1)
-FIRST_DAY_CURRENT_YEAR = lambda: date.today().replace(day=1) + relativedelta(month=1)
-FIRST_DAY_NEXT_YEAR = lambda: date.today().replace(day=1) + relativedelta(month=1, years=+1)
+TODAY = lambda: date.today() + timedelta(days=0)# pylint: disable=unnecessary-lambda-assignment
```

Comment:
maybe just add a single `# pylint: disable=unnecessary-lambda-assignment` to the top of the file